.github/workflows/ci.yml

```diff
@@ -20,7 +21,6 @@ jobs:
         ruby-version: ${{ matrix.ruby }}
     - name: Build
       run: |
-        gem install bundler
```

Comment:
Was causing failures on ruby 2.7. Doesn't seem to be needed anyways, as the prior step installs bundler.